packages/api/src/external/index.ts

```diff
@@ -12,4 +13,5 @@ export const HL7OID = "2.16.840.1.113883";
 
 export const MedicalDataSourceOid: Record<MedicalDataSource, string> = {
   [MedicalDataSource.COMMONWELL]: `${HL7OID}.3.3330`,
+  [MedicalDataSource.CAREQUALITY]: `${HL7OID}.3.3330`,
```

Comment:
this is CW http://oid-info.com/get/2.16.840.1.113883.3.3330

we need to get the one for CQ if it exists

Reply:
Doesnt seem like they do will remove